### ISSUE TITLE
Building Rmd file with a missing graphics file triggers a tlmgr search

### ISSUE THREAD

**Issue by jsta** [yihui/tinytex#94]
I am trying to build a `pinp` project with `tinytex`. As a MWE, if I try to run `rmarkdown::render` on the following file:

https://github.com/eddelbuettel/pinp/blob/master/inst/rmarkdown/templates/pdf/skeleton/skeleton.Rmd

where line [182](https://github.com/eddelbuettel/pinp/blob/master/inst/rmarkdown/templates/pdf/skeleton/skeleton.Rmd#L182) is set to `eval = FALSE` (so that the graphics file `densities` does not exist), I get a `tlmgr` error that is very confusing:

```
tlmgr search --file --global 'densities'

Error: Failed to compile manuscript.tex. See manuscript.log for more info.
In addition: Warning message:
In parse_packages(logfile, quiet = c(TRUE, FALSE, FALSE)) :
  Failed to find a package that contains densities
```

**Comment by yihui** [comment]
I understand you may be confused by `tlmgr search --file --global 'densities'`, but the actual error message says:

> Error: Failed to compile manuscript.tex. See manuscript.log for more info.

So you need to see `manuscript.log`.

When a file cannot be found, **tinytex** is not smart enough to know whether it is a LaTeX package file or a graphics file. It will try to see if it is a missing LaTeX package first.

Anyway, since you consciously set `eval = FALSE` and got this error, I guess that somehow implies `eval = FALSE` is the culprit.

I don't know what to do in this case. Sorry!

BTW, my error message is actually this:

```
tlmgr search --file --global '/densities'
! LaTeX Error: File `densities' not found.

Error: Failed to compile test.tex. See test.log for more info.
In addition: Warning message:
In parse_packages(logfile, quiet = c(TRUE, FALSE, FALSE)) :
  Failed to find a package that contains densities
```

i.e. the actual LaTeX error message is

```
! LaTeX Error: File `densities' not found.
```

I'm not sure why this line was absent in your log.

**Comment by jsta** [comment]
Thanks for looking into this! `eval=FALSE` was a device only to create a MWE in lieu of my much more complicated project where the graphics file was simply missing. Anyway, sounds like " tinytex is not smart enough to know whether it is a LaTeX package file or a graphics file" is not an oversight or a trivial fix. I opened the issue because I spent a good amount of time trying to set `latexmk(install_packages=FALSE`.
